package.json

```diff
@@ -24,7 +24,7 @@
     "flow:install-types": "flow-typed install --ignoreDeps dev",
     "flow:install-types:all": "flow-typed install",
     "postinstall": "jetifier",
-    "trigger-pipeline": ".circleci/trigger-pipeline.sh",
+    "trigger-pipeline": ".circleci/trigger-pipeline",
```

Comment:
What  does `trigger-pipeline` do?
Also I'm pretty sure, this won't work on windows, right? Probably it would make sense to "translate" trigger-pipeline to a node-js script?

Reply:
Created an issue :)